test/tizen_capi/unittest_tizen_capi_dataset.cpp

```diff
@@ -120,9 +120,20 @@ TEST(nntrainer_capi_dataset, create_destroy_06_n) {
 }
 
 /**
- * @brief Neural Network Dataset Create / Destroy Test (positive test )
+ * @brief Neural Network Dataset Create / Destroy Test (negative test)
```

Comment:
Can you delete this test cause I made same one

Reply:
okay